lib/ain-evm/src/core.rs

```diff
@@ -160,8 +160,9 @@ impl EVMCoreService {
     pub fn validate_raw_tx(
         &self,
         tx: &str,
-        with_gas_usage: bool,
-    ) -> Result<(SignedTx, u64), Box<dyn Error>> {
+        call_tx: bool,
+        context: u64,
+    ) -> Result<(SignedTx, U256, u64), Box<dyn Error>> {
```

Comment:
Can we switch the result to a struct please?

Reply:
Fixed in commit [e92a216](https://github.com/DeFiCh/ain/pull/2180/commits/e92a2160f53edb1f0802324cf98ea2a1c71e1f1e)